simulariumio/data_objects/camera_data.py

```diff
@@ -57,38 +58,31 @@ def __init__(
         self.fov_degrees = fov_degrees
 
     @classmethod
-    def from_buffer_data(cls, buffer_data: Dict[str, Any]):
-        """ """
-        camera_default = (
-            buffer_data["trajectoryInfo"]["cameraDefault"]
-            if "cameraDefault" in buffer_data["trajectoryInfo"]
-            else None
-        )
-        if camera_default is None:
+    def from_dict(cls, camera_default: Dict[str, Any]):
```

Comment:
this is probably `camera_info` rather than `camera_default`?

Reply:
yeah, you're right :)